### ISSUE TITLE
Алл чат понимают все расы

### ISSUE THREAD

**Issue by Zevs2018** [goodwinxp/Yorozuya#111]
Можешь сделать модуль алл чата чтобы понимали все расы ?

**Comment by goodwinxp** [comment]
Что значит понимали? all чат только в рамках 1 расы. Или ты про белый чат? Так на то есть поюк

**Comment by QuiX43** [comment]
Наверно,он имел ввиду общий /all чат модуль.

**Comment by goodwinxp** [comment]
@Zevs2018, комментарии будут?

**Comment by Zevs2018** [comment]
Да, (общий /all чат модуль) ещё можно ли сделать что бы в краговых шахтах (на пример раса беллато не могла видеть /all чат расы кора)

**Comment by goodwinxp** [comment]
Как по мне, так идея с общим /all чат - бредовая и нужно ограничиться поюком, т.к.
- всё таки ЛОР игры подразумевает враждебность рас, а не "мир, дружба, жвачка"
- иногда координирование действий идёт через /all чат, причём не только ЧВ, но и пб, и какая-то стратегическая инфа по типу разлока той или иной карты

так что даже если "вдруг" это наберёт 10 голосов "за", то делать я это не буду, но рассмотрю pull request если кто-то другой сделает

**Comment by fortuner92** [comment]
А если я предложу сделать актуальный модуль?
Уничтожив чужой чип на войне, будет падать предмет прослушка.
Сделать его допингом.
Тем самым будут партизанские методы войны.
